lib/platform/systemd_service.rb

```diff
@@ -21,6 +21,11 @@ def self.configured?(service_name, host)
         host.exec_admin("systemctl is-active '#{service_name}'")[:success]
       end
 
+      def self.enabled?(service_name, host)
+        result = host.exec_admin("systemctl is-enabled '#{service_name}'")
+        result[:success] && (result[:stdout].to_s.include? "enabled")
+      end
```

Comment:
@akostadinov  I added this for checking if openvswitch is enabled on the host.  I assume this is okay to use for any random systemd service.